### PR TITLE
Dsh/week 18

### PR DESCRIPTION
**Pull request by dHelmgren**
There's still work to do here I think, but it's at least a solid first pass at how y'all might approach this.

### CODE REVIEW

**Comment by CheezItMan** [comment]
I made a few improvements to the answers using regex, otherwise I think this looks ok.

File: config.yaml [hunk 1]
```diff
@@ -102,11 +102,19 @@ Standards:
...
       - success criteria
     ContentFiles:
+      - 
+        Type: Lesson
+        UID: 5ae06a42-94f2-4927-8831-12987d7538fb
+        Path: /03-react-week-2/apis-get.md
+      -
+        Type: Lesson
+        UID: 0e9e569d-bd26-4c57-8b2d-77ccb15fcde9
+        Path: /03-react-week-2/apis-post.md
```
Comment (spitsfire) [inline config.yaml:117]: should these should go after advanced forms based on the checklist Jared laid out?

**Review by spitsfire — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (jmaddox19) [reply config.yaml:117]: Yes, definitely. Thank you for catching that! I would love for us to be careful about ordering the lessons in the order we plan for students to consume them.

Comment (CheezItMan) [reply config.yaml:121]: I reordered it.

**Pull request merged**